interface/billing/billing_process.php

```diff
@@ -149,14 +151,25 @@ function send_batch()
         fwrite($fh, $bat_content);
         fclose($fh);
     }
-    header("Pragma: public");
-    header("Expires: 0");
-    header("Cache-Control: must-revalidate, post-check=0, pre-check=0");
-    header("Content-Type: application/force-download");
-    header("Content-Disposition: attachment; filename=$bat_filename");
-    header("Content-Description: File Transfer");
-    header("Content-Length: " . strlen($bat_content));
-    echo $bat_content;
+    if (!$GLOBALS['enable_clearinghouse']) {
+        header("Pragma: public");
+        header("Expires: 0");
+        header("Cache-Control: must-revalidate, post-check=0, pre-check=0");
+        header("Content-Type: application/force-download");
+        header("Content-Disposition: attachment; filename=$bat_filename");
+        header("Content-Description: File Transfer");
+        header("Content-Length: " . strlen($bat_content));
+        echo $bat_content;
+    } else {
+        $clearinghouse = new ClearingHouse();
+        $response = $clearinghouse->sendBilling($bat_filename);
+        if ($response == 'Sucesss') {
```

Comment:
Success

Reply:
oh  wait, disregard that since not part of front end (ie. does not need to be spelled correctly)